frb/data/analysis/CIGALE/DEIMOS_r.dat

```diff
@@ -1,4 +1,5 @@
 #DEIMOS_r
+#photon
```

Comment:
does this need to be done for every file in the folder?

Reply:
I checked the other files and the DEIMOS filters were the only ones lacking this metadata.